webpack-plugin/package-lock.json

```diff
@@ -1,8 +1,1788 @@
 {
 	"name": "monaco-editor-webpack-plugin",
 	"version": "7.0.0",
-	"lockfileVersion": 1,
+	"lockfileVersion": 2,
 	"requires": true,
+	"packages": {
+		"": {
+			"name": "monaco-editor-webpack-plugin",
+			"version": "7.0.0",
+			"license": "MIT",
+			"dependencies": {
+				"json5": "2.2.3",
+				"loader-utils": "^2.0.4"
+			},
+			"devDependencies": {
+				"css-loader": "^6.5.1",
+				"file-loader": "^6.2.0",
+				"glob": "^7.2.0",
+				"monaco-editor": "^0.31.0",
+				"style-loader": "^3.3.1",
+				"typescript": "^4.4.4",
+				"webpack": "^5.64.1",
+				"webpack-cli": "^4.9.1"
+			},
+			"peerDependencies": {
+				"monaco-editor": ">= 0.31.0",
+				"webpack": "^4.5.0 || 5.x"
+			}
+		},
+		"node_modules/@discoveryjs/json-ext": {
+			"version": "0.5.5",
```

Comment:
Most of the changes on this file are weird. Can you double check (reverting and doing a npm install again) if they are correct?